src/main/java/uk/gov/companieshouse/accounts/filing/service/company/CompanyServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.companieshouse.accounts.filing.service.company;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.accounts.filing.model.AccountsFilingEntry;
+import uk.gov.companieshouse.accounts.filing.model.CompanyResponse;
+import uk.gov.companieshouse.accounts.filing.repository.AccountsFilingRepository;
+import uk.gov.companieshouse.logging.Logger;
+
+import java.util.Map;
+
+@Service
+public class CompanyServiceImpl implements CompanyService{
+
+    private final Logger logger;
+    private final AccountsFilingRepository accountsFilingRepository;
+    @Autowired
+    public CompanyServiceImpl(
+            Logger logger,
+            AccountsFilingRepository accountsFilingRepository) {
+        this.logger = logger;
+        this.accountsFilingRepository = accountsFilingRepository;
+    }
+
+    @Override
+    public CompanyResponse saveCompanyNumberAndTransactionId(String companyNumber, String transactionId) throws RuntimeException {
+        var response = new CompanyResponse(accountsFilingRepository.save(new AccountsFilingEntry(null, null, null,null, companyNumber, transactionId)).getAccountFilingId());
```

Comment:
This line is a bit long maybe
```suggestion
        var response = new CompanyResponse(accountsFilingRepository.save(
            new AccountsFilingEntry(null, null, null,null, companyNumber, transactionId)).getAccountFilingId());
```